packages/app-page-builder/src/render/plugins/elements/block/Block.tsx

```diff
@@ -8,6 +8,19 @@ import ElementAnimation from "../../../components/ElementAnimation";
 import { Interpolation } from "@emotion/core";
 import { PageBuilderContext } from "../../../../contexts/PageBuilder";
```

Comment:
When you see this type of imports, please try rewriting them like this:
```
import { PageBuilderContext } from "~/contexts/PageBuilder";
```

Every package we have, has a `~/` mapped to the `src/` folder of the package, so you can use that for nicer paths.